docs/dashboards-classic/get-started/create-dashboard.md

```diff
@@ -4,7 +4,7 @@ title: Create a Dashboard
 ---
 
 :::caution
-We strongly recommend using our [Dashboards (New)](/docs/dashboards-new), which allows you to analyze metrics and log data on the same dashboard in a streamlined user experience. Sumo Logic will cease updates and support for Classic Dashboards.
+We strongly recommend using our [Dashboards (New)](/docs/dashboards), which allows you to analyze metrics and log data on the same dashboard in a streamlined user experience. Sumo Logic will cease updates and support for Classic Dashboards.
```

Comment:
Do you need to remove "(New)" here?

Reply:
I have updated the sentence without this (New) word now.